### PR TITLE
Feat: text-utilities, navigation, list

### PR DESCRIPTION
**Pull request by yashpatel1797**
Added - Text utilities, navigation, lists components.
[live link](https://vibrant-jang-276ddd.netlify.app/docs.html)

### CODE REVIEW

File: components/List/list.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (TejasShekar) [inline components/List/list.html]: Change the title as per your document format for a component

File: components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
+.nav-bar {
+  display: flex;
+  align-items: center;
+  background-color: var(--color-yellow);
+}
+.nav-section {
+  margin: 0rem auto 0rem 4rem;
```
Comment (TejasShekar) [inline components/Navigation/navigation.css:7]: you can use display flex here

File: components/Navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (TejasShekar) [inline components/Navigation/navigation.html]: 1. Change title to the component name
2. Add author meta tag

**Review by kotesh-arya — APPROVED** [review]
Looks clean and good for a merge 👍🏻

**Review by TejasShekar — COMMENTED** [review]
Kindly look into my suggestions and implement it in other files as well. Rest looks good to me. 👍

File: components/List/list.html (hunk 1 above)
Comment (yashpatel1797) [reply components/List/list.html]: thanks for this I forgot to change in almost every file will do that before merging.

File: components/Navigation/navigation.html (hunk 3 above)
Comment (yashpatel1797) [reply components/Navigation/navigation.html]: ok

**Pull request merged**